modules/primer-select-menu/lib/select-menu.scss

```diff
@@ -0,0 +1,563 @@
+// Needs refactoring
+// stylelint-disable selector-no-qualifying-type, selector-max-type
+// Styles for the click target button for the select-menu
+.select-menu-button {
+  // Caret
+  &::after {
+    display: inline-block;
+    width: 0;
+    height: 0;
+    vertical-align: -2px;
+    content: "";
+    border: 4px solid;
+    border-right-color: transparent;
+    border-bottom-color: transparent;
+    border-left-color: transparent;
+  }
+
+  &.icon-only {
+    padding-left: 7px;
+  }
+
+  &.primary {
+    &::after {
+      border-top-color: $white;
+
+      &:active {
```

Comment:
I realize this is an old and closed PR, but I wanted to bring attention to this selector because it's still in use on github.com. The order of these selectors is incorrect and Chrome & Safari don't load this rule at all.

If you switch the order from `:after:active` to `:active:after` then it gets loaded and applied. I came across this while working on GitHub-Dark because one of our green background rules wasn't being applied to other elements and it was because this selector was part of that rule.

You can see an example of this at https://codepen.io/xt0rted/pen/OYLbeJ where the lime background doesn't get applied to either element but does if you switch the selector order.